src/commands/lambda/functions/commons.ts

```diff
@@ -418,23 +418,26 @@ export const isLayerRuntime = (runtime: string): runtime is LayerKey => LAYER_LO
 
 export const sentenceMatchesRegEx = (sentence: string, regex: RegExp) => sentence.match(regex)
 
-export const updateLambdaFunctionConfigs = async (
+export const updateLambdaFunctionConfig = async (
   lambda: Lambda,
   cloudWatch: CloudWatchLogs,
-  configs: FunctionConfiguration[]
+  config: FunctionConfiguration
 ) => {
-  const results = configs.map(async (c) => {
-    if (c.updateRequest !== undefined) {
-      await lambda.updateFunctionConfiguration(c.updateRequest).promise()
+  try {
+    if (config.updateRequest !== undefined) {
+      await lambda.updateFunctionConfiguration(config.updateRequest).promise()
     }
-    if (c.logGroupConfiguration !== undefined) {
-      await applyLogGroupConfig(cloudWatch, c.logGroupConfiguration)
+    if (config.logGroupConfiguration !== undefined) {
+      await applyLogGroupConfig(cloudWatch, config.logGroupConfiguration)
     }
-    if (c.tagConfiguration !== undefined) {
-      await applyTagConfig(lambda, c.tagConfiguration)
+    if (config.tagConfiguration !== undefined) {
+      await applyTagConfig(lambda, config.tagConfiguration)
     }
-  })
-  await Promise.all(results)
+  } catch (e) {
+    if (e instanceof Error) {
```

Comment:
Hmmm, is there a case where `e` would not be an instance of an `Error`? I'm not sure of one.

Also either way, we don't need to create a new Error here, and likely don't need the try-catch block at all if we're not handling any error.

Reply:
I was doing it to type guard errors, since every error is of type `unknown`. But thank you for catching it! I don't know why I was raising a `new` error. I think the current state needs to be changes to address those comments. Thank you AJ.